tests/customer-resource-show-visibility-test.js

```diff
@@ -98,12 +184,76 @@ describeApplication('CustomerResourceShowVisibility', () => {
       });
     });
 
-    it('displays the hidden/reason section', () => {
-      expect(CustomerResourceShowPage.isHidden).to.be.true;
+    it('the hide/show toggle with hide', () => {
```

Comment:
Alternate suggestion for this label: `displays the visibility toggle as switched to on`.